validation-test/stdlib/FixedPointConversion.swift.gyb

```diff
@@ -88,15 +102,120 @@ FixedPointConversionTraps.test("${Other}To${Self}Conversion/dest=${testValue}")
 FixedPointConversionFailure.test("${Other}To${Self}Conversion/dest=${testValue}") {
   // Test that we check if we return nil when an integer would be truncated in conversion.
   let input = get${Other}(${testValue})
-  var result = ${Self}(exactly: input)
-  expectEqual(nil, result)
-  _blackHole(result)
+  expectEmpty(${Self}(exactly: input))
 }
 %       end
 
 %     end # for testValue in ...
 %   end # for in all_integer_types (Other)
+
+%   for other_type in all_floating_point_types():
+%    Other = "Float" + str(other_type.bits)
```

Comment:
Could you change this code to use 2-space indentation, like we do in other gyb code?